src/examples/java/io/nats/examples/chaosTestApp/ConnectableConsumer.java

```diff
@@ -37,7 +37,7 @@ public abstract class ConnectableConsumer implements ConnectionListener {
     protected String durableName;
     protected String label;
 
-    public ConnectableConsumer(CommandLine cmd, String initials, ConsumerKind consumerKind) throws IOException, InterruptedException, JetStreamApiException {
+    public ConnectableConsumer(CommandLine cmd, String initials, ConsumerKind consumerKind) throws IOException, InterruptedException {
```

Comment:
Non client production code, only if already changing or fixing the class. BUT... This could be cleaned up if you were already working on this and in it's own PR.